src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working");

```

Comment:
The message should indicate what type of `Machine` is working right now. Consider making the message more informative, such as 'Truck is starting to work'.